### PR TITLE
Add Arabic language

### PR DESCRIPTION
**Pull request by abutmim**

### CODE REVIEW

File: xdrip/Storyboards/‏‏ar.lproj/SettingsViews.strings [hunk 1]
```diff
@@ -1,79 +1,79 @@
...
+"calenderId" = "التقويم للاستخدام؟";
+"displayTrendInCalendarEvent" = "عرض الإتجاه؟";
+"displayUnitInCalendarEvent" = "عرض الوحدة؟";
+"displayDeltaInCalendarEvent" = "عرض الفرق؟";
+"infoCalendarAccessDeniedByUser" = "لقد رفضت الوصول إلى التقويم الخاص بك مسبقًا.\n\nلتمكينه ، انتقل إلى إعدادات جهازك ثم الخصوصية ثم التقويم ثم قم بتمكينه.";
+"infoCalendarAccessRestricted" = "لا يمكنك منح إذن لـ xDrip4iOS للوصول إلى التقويم الخاص بك. قد يكون هذا بسبب القيود النشطة مثل وجود أدوات الرقابة الأبوية.";
+"writeTraceToFile" = "كتابة التتبع إلى ملف؟";
+"sectionTitleTrace" = "تتبع";
+"sendTraceFile" = "إرسال ملف التتبع";
+"describeProblem" = "اشرح سبب حاجتك لإرسال ملف التتبع بأكبر قدر ممكن من التفاصيل. إذا كنت قد ذكرت بالفعل مشكلتك في مجموعة دعم الفيسبوك "xDrip4iOS"، اذكر اسم الفيسبوك الخاص بك في البريد الإلكتروني";
```
Comment (JohanDegraeve) [inline xdrip/Storyboards/‏‏ar.lproj/SettingsViews.strings:77]: when quoting "xdrip4iOS"  can you replace the double quote by single quote ?

**Comment by abutmim** [comment]
Done

File: xdrip/Storyboards/‏‏ar.lproj/SettingsViews.strings (hunk 1 above)
Comment (abutmim) [reply xdrip/Storyboards/‏‏ar.lproj/SettingsViews.strings]: did it

**Comment by JohanDegraeve** [comment]
Before adding new translations in the future,
please do a sync first of your fork, i think it should work with "git fetch upstream"

**Pull request merged**